src/app/archive/create_schema.sql

```diff
@@ -58,6 +74,48 @@ CREATE TABLE internal_commands
 , UNIQUE (hash,type)
 );
 
+/* import supporting Snapp-related tables */
+\i snapp_tables.sql
+
+CREATE TABLE snapp_fee_payers
+( id                       serial           PRIMARY KEY
+, body_id                  int              NOT NULL REFERENCES snapp_party_body(id)
+, nonce                    bigint           NOT NULL
+);
+
+/* encode a list of party's: the list_id identifies the list, the list_index indicates the order */
+CREATE TABLE snapp_other_parties
+( list_id                  int              NOT NULL
+, list_index               int              NOT NULL
+, party_id                 int              NOT NULL REFERENCES snapp_party(id)
+, PRIMARY KEY (list_id,list_index)
+);
+
+/* NULL convention -- see comment at start of snapp_tables.sql */
+CREATE TABLE snapp_protocol_states
+( id                               serial                         NOT NULL PRIMARY KEY
+, snarked_ledger_hash_id           int                            REFERENCES snarked_ledger_hashes(id)
+, snarked_next_available_token_id  int                            REFERENCES snapp_bounded_token_id(id)
```

Comment:
Can we change `bounded` to `interval`/`bounds` or something? To me this suggests the table has bounded values and not the bounds.  For example `snapp_timestamp_interval`/`snapp_global_slot_bounds`

Reply:
Done.